### PR TITLE
[TH2-1128] Migrate to th2 storage 2.4

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
* Defined version for bom and cradle explicitly

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -107,7 +107,7 @@ allprojects {
                 desc = project.version
                 released  = new Date()
             }
-        }
+            }
```
Comment (OptimumOpium) [inline build.gradle:110]: How has it happened?)

File: build.gradle [hunk 2]
```diff
@@ -127,13 +127,13 @@ test {
 }
 
 dependencies {
-    api platform('com.exactpro.th2:bom:2.3.+')
-    api 'com.exactpro.th2:cradle-core'
+    api platform('com.exactpro.th2:bom:2.9.1')
+    api 'com.exactpro.th2:cradle-core:2.4.0'
```
Comment (OptimumOpium) [inline build.gradle:131]: Maybe we should extract the cradle version to a variable?

**Review by OptimumOpium — APPROVED** [review]

File: build.gradle (hunk 2 above)
Comment (Nikita-Smirnov-Exactpro) [reply build.gradle:131]: I think no because this project has only one artifact related to cradle

Comment (OptimumOpium) [reply build.gradle:131]: And what about `implementation 'com.exactpro.th2:cradle-cassandra:2.4.0'`?

Comment (Nikita-Smirnov-Exactpro) [reply build.gradle:131]: O, skipped it, thanks

**Pull request merged**